src/Tasks/CultureInfoCache.cs

```diff
@@ -18,10 +18,12 @@ namespace Microsoft.Build.Tasks
     internal static class CultureInfoCache
     {
         private static readonly HashSet<string> ValidCultureNames;
+        private static readonly HashSet<string> InvalidCultureNames;
```

Comment:
```suggestion
        private static readonly HashSet<string> KnownInvalidCultureNames;
```

? I was initially a bit confused since this isn't an exhaustive list.